sass/util/_width.scss

```diff
@@ -0,0 +1 @@
+.mw8 { max-width: 20rem; }
```

Comment:
why the `8` in classname? maybe make it more explicit tied to the length, i.e., `mw20`?